src/ui/dom/dom.js

```diff
@@ -95,19 +95,19 @@ export default class DOM {
   /**
    * createEle will create a {HTMLElement} and apply the properties attributes through an object provided.
    * @param {string} el The element `tag` name to construct
-   * @param {Object} opts_data Optional attributes to apply to the new HTMLElement
+   * @param {Object} optsData Optional attributes to apply to the new HTMLElement
    */
-  static createEl (el, opts_data = {}) {
+  static createEl (el, optsData = {}) {
```

Comment:
this change was originally from when I tried to update some of the eslint config dependencies we inherit from, from when I tried to update our eslint- related dependencies without using npm overrides

I think we don't need it anymore but I think I prefer having opts_data in camel case anyways